lerna.json

```diff
@@ -1,12 +1,17 @@
-{
-  "lerna": "2.8.0",
-  "packages": [
-    "packages/*"
-  ],
-  "command": {
-    "bootstrap": {
-      "npmClientArgs": ["--no-package-lock"]
-    }
-  },
-  "version": "independent"
-}
+{
+  "lerna": "2.8.0",

```

Comment:
This key does not seem to have any meaning anymore, you can remove it.

Reply:
The key was removed.